<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>611</width>
     <height>61</height>
    </size>
   </property>
   <property name="title">
    <string>串口配置</string>
   </property>
   <widget class="QComboBox" name="comboBox_serialPort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_serialBaud">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_openSerial">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_radioConfig">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>端口:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>611</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>611</width>
     <height>61</height>
    </size>
   </property>
   <property name="title">
    <string>电台配置</string>
   </property>
   <widget class="QComboBox" name="comboBox_radioLanguage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>中文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_radioBrand">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>通用</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中海达</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中纬</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拓普康</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宾得</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>光谱</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天宝</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_radioYear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_radioMonth">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_radioNumber">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>语言:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>品牌:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>月:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>序列号:</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_serialRxBuf">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>511</width>
     <height>291</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>101</width>
     <height>281</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>281</height>
    </size>
   </property>
   <property name="title">
    <string>写操作</string>
   </property>
   <widget class="QPushButton" name="pushButton_cleanRecv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除缓存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_radioWriteNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>写号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_radioSave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
